internal/usergroup/add_linux.go

```diff
@@ -129,38 +131,44 @@ func createSubordinateRanges(name string) error {
 }
 
 func findNextUIDRange() (int, error) {
-	ranges, err := parseSubuid("ALL")
+	ranges, err := user.CurrentUserSubUIDs()
```

Comment:
I had a quick peek, but I think we can make these changes before splitting the packages, that way we don't have to re-do them when adding back the aliases.